tests/test.sh

```diff
@@ -105,7 +105,10 @@ DIA="docker run $DOCKER_OPT --entrypoint dia dalibo/pandocker:$TAG --verbose"
 $DIA $IN/db.dia --export $OUT/db.svg
 
 # 16. Inline PDF output
-cat $IN/markdown_de.md | docker run --rm -i dalibo/pandocker:$TAG --to=pdf --pdf-engine=xelatex > $OUT/markdown_de.inline.pdf
+cat $IN/markdown_de.md | docker run --rm -i dalibo/pandocker:$TAG --to=latex --pdf-engine=xelatex > $OUT/markdown_de.inline.pdf
```

Comment:
The `to=pdf` option is a brand new addition (since pandoc 2.8 I think). It allows to user to produce a PDF output without specifying a `.pdf` suffix in the `-o` option. Therefore it is useful when we want to pass the PDF content through pipes and get a new inline stream without mounting a docker volume.

With the change you've made, the `markdown_de.inline.pdf` would actually contain the latex version of the document.